parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -376,6 +381,15 @@ pub type AssetFeeAsExistentialDepositMultiplierFeeCharger = AssetFeeAsExistentia
 	TrustBackedAssetsInstance,
 >;
 
+/// Cases where a remote origin is accepted as trusted Teleporter:
+/// - teleportation of KSM from the parent Relay Chain and sibling system parachains; and
+/// - teleportation of sibling parachain's assets (as ForeignCreators).
+pub type TrustedTeleporters = (
+	NativeAsset,
+	IsForeignConcreteAsset<FromSiblingParachain<parachain_info::Pallet<Runtime>>>,
+	ConcreteNativeAssetFromSiblingSystemParachain,
```

Comment:
This is strictly more restrictive than `NativeAsset`, shouldn't we remove `NativeAsset` then as trusted teleporters?

Reply:
It'd also need to include the Relay Chain, so add `Multilocation { parents: 1, interior: Here } => true,` to the match, and then it could be `ConcreteNativeAssetFromSystem`.